generators/app/templates/src/app/screens/Auth/screens/Login/index.test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import * as redux from 'react-redux';
+import { render, fireEvent, waitFor, waitForElementToBeRemoved } from 'react-native-testing-library';
+import Login from '@authScreens/Login';
+
+const INVALID_EMAIL = 'hello';
+const VALID_EMAIL = 'email@email.com';
+const VALID_PASSWORD = 'HelloWord1234';
+
+describe('<Login />', () => {
+  test('Invalid inputs', async () => {
+    const { getByText, getAllByText, getByTestId } = render(<Login />);
+
+    const submitButton = getByText('Iniciar sesión');
+    const emailInput = getByTestId('Email');
+
+    fireEvent.press(submitButton);
+    await waitFor(() => getAllByText('Este campo es obligatorio'));
+    expect(getAllByText('Este campo es obligatorio')).toHaveLength(2);
+
+    fireEvent.changeText(emailInput, INVALID_EMAIL);
+    fireEvent.press(submitButton);
+
+    await waitFor(() => getByText('El formato del mail es inválido'));
+
+    fireEvent.changeText(emailInput, VALID_EMAIL);
+    fireEvent.press(submitButton);
+
+    await waitForElementToBeRemoved(() => getByText('El formato del mail es inválido'));
```

Comment:
`waitForElementToBeRemoved` waits until his argument vanish
this would fail if the element wasn't already there or if doesn't disappear (it would timeout)